jonathanmusila/Questioner-Django
# #165657045  set travis

#### What does this PR do?
sets up Travis for testing environment

#### Description of Task to be completed?
- [x] setup Travis file

#### How should this be manually tested?\
- go to the repo 
`https://github.com/jonathanmusila/Questioner-Django`

- check build badge and click on it

#### What are the relevant pivotal tracker stories?
#165657045
